driver/src/test/java/org/neo4j/driver/v1/integration/TransactionAsyncIT.java

```diff
@@ -1157,7 +1157,7 @@ public void shouldFailToCommitWhenRunFailureIsConsumed()
         }
         catch ( ClientException e )
         {
-            assertThat( e.getMessage(), startsWith( "Transaction rolled back" ) );
+            assertThat( e.getMessage(), startsWith( "No current transaction to commit." ) );
```

Comment:
This message change is quite weird. It exposes an implementation detail that driver resets a connection after an error. Do you think it is okay?